packages/composables/__tests__/useCart/useCart.spec.ts

```diff
@@ -99,9 +100,16 @@ describe('[vendure-composables] useCart', () => {
     expect(response).toEqual({
       updatedCart: 'remove coupon code'
     });
-    expect(context.$vendure.api.removeCouponCode).toBeCalledWith(
+    expect(context.$vendure.api.removeCartCoupon).toBeCalledWith(
       { couponCode: '1' },
       customQuery
     );
   });
+
+  it('checks if product is in cart', async () => {
+    const { isInCart } = useCart() as any;
+    const response = await isInCart(context, { currentCart: mockedCart, product: { _id: '4' } });
+
+    expect(response).toEqual(true);
+  });
```

Comment:
Please provide another test case where you will be checking is product in the cart when the cart is empty